src/plugins/data-ajax/data-ajax.js

```diff
@@ -130,6 +131,8 @@ $document.on( "timerpoke.wb " + initEvent + " " + updateEvent + " ajax-fetched.w
 
 				//Resets the initial jQuery caching setting
 				jQuery.ajaxSettings.cache = jQueryCaching;
+
+				$elm.trigger( contentUpdatedEvent, { type: ajaxType, content: content } );
```

Comment:
I'm wondering if the type overwrites the event name. It probably is safer like: `$elm.trigger( { type: contentUpdatedEvent, content: content } );`